doc/dev/cephadm/developing-cephadm.rst

```diff
@@ -259,21 +259,25 @@ Setup
 In order to setup Cephadm's box run::
 
   cd src/cephadm/box
-  sudo ln -sf "$PWD"/box.py /usr/bin/box
-  sudo box -v cluster setup
+  ./box.py -v cluster setup
 
-.. note:: It is recommended to run box with verbose (-v).
+.. note:: It is recommended to run box with verbose (-v) as it will show the output of
+					shell commands being run.
 
 After getting all needed images we can create a simple cluster without osds and hosts with::
 
-  sudo box -v cluster start
+  ./box.py -v cluster start
 
 If you want to deploy the cluster with more osds and hosts::
   # 3 osds and 3 hosts by default
   sudo box -v cluster start --extended
   # explicitly change number of hosts and osds
   sudo box -v cluster start --extended --osds 5 --hosts 5
 
+.. warning:: Osds are still not supported in the box implementation with podman. It is
+						 work in progress.
```

Comment:
s/Osds/OSDs/

I'm a bit uncertain how the cluster works without OSDs, or are OSDs deployed with something other than podman?

Reply:
OSDs are still not being deployed but there is a branch owned by @zmc where he did create OSDs with ceph-volume. There are certain things that we can test without OSDs in the dashboard team and the mgr in general, but yeah, we have to add those.